components/layout/header/index.tsx

```diff
@@ -9,10 +9,11 @@ import NavBar from './nav-bar';
 const Header: FC = () => (
   <>
     <Box
-      mb="0!important"
+      mb="0 !important"
```

Comment:
please, delete this `mb="0 !important"`, it doesn't work and do not affect on the frontend